doc/internal/man3/ossl_namemap_new.pod

```diff
@@ -40,6 +41,10 @@ new B<OSSL_NAMEMAP>.
 This is suitable to use when the B<OSSL_NAMEMAP> is embedded in other
 structures, or should be independent for any reason.
 
+ossl_namemap_empty() checks if the given B<OSSL_NAMEMAP> is empty or
+not.
+This can be useful for pre-population purposes.
```

Comment:
I'd omit this line, it's describing a use rather than the function.

Reply:
Okie